### PR TITLE
Random walk fix

### PR DESCRIPTION
**Pull request by ColinEberhardt**
It was outputting steps + 1 values

### CODE REVIEW

File: tests/data/random/walkSpec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+describe('walk', function() {
+
+    var element;
```
Comment (chrisprice) [inline tests/data/random/walkSpec.js]: Copy/paste left over?

File: tests/data/random/walkSpec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    beforeEach(function() {
+        element = document.createElement('svg');
```
Comment (chrisprice) [inline tests/data/random/walkSpec.js]: Ditto?

**Comment by chrisprice** [comment]
Referencing #832 as this change should be reflected there as well.

**Comment by ColinEberhardt** [comment]
@chrisprice  Thanks for spotting, I've removed the leftovers

File: tests/data/random/walkSpec.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        var walk = fc.data.random.walk()
+            .steps(5);
+
+        var walkData = walk(10);
+
+        // this test data was generted from the random walk at the following point:
+        // b0ee0a397a6ecc4cf60663d5b51af15de3534ef1
+        // and is assumed correct for the purposes of further refactor.
+        expect(walkData)
+            .toEqual([10, 10.563051249613967, 10.862390962256633, 11.473999242766038, 11.799153741691457]);
```
Comment (tlsim) [inline tests/data/random/walkSpec.js]: Should the output include the initial price? If so, I'd expect the output to have `steps` + 1 length if `steps` is the number of price increments. E.g. for steps = 1, period = T, Initial price S<sub>0</sub>, the output should be either [S<sub>0</sub>, S<sub>T</sub>] or [S<sub>T</sub>], but not [S<sub>0</sub>]. The current documentation for `steps` is a bit misleading here.

**Comment by ColinEberhardt** [comment]
From discussion with @tlsim - the API for random walk is intended to represent the concept of dividing a period of time into `n` subdivisions. As a result, the output will be `n + 1` points.

**Pull request merged**